cloud-startup.sh

```diff
@@ -20,28 +31,86 @@ done
 
 echo 'Database connection successful'
 
-# Run database migrations
+# Create database if it doesn't exist
+# This ensures the application's database exists before proceeding
+mysql -h "$MYSQL_SERVER" -u "$MYSQL_USER" -p"$MYSQL_PASSWORD" <<-EOSQL
+    CREATE DATABASE IF NOT EXISTS ${MYSQL_DATABASE};
+    USE ${MYSQL_DATABASE};
+EOSQL
+
+if ! mysql -h "$MYSQL_SERVER" -u "$MYSQL_USER" -p"$MYSQL_PASSWORD" -e "USE ${MYSQL_DATABASE}"; then
+    echo "Failed to create/access database ${MYSQL_DATABASE}"
+    exit 1
+fi
+
+echo "Database ${MYSQL_DATABASE} ready"
+
+# Run database migrations using alembic
+# This ensures all database tables and schemas are up to date
+cd /app
 alembic upgrade head
 
-# check backend input parameters
+# Initialize admin user and organization
```

Comment:
Although the content is good for the time being, it is assumed that it is appropriate to make this process a separate script so that it can be run in the local development environment.

Reply:
This script is specifically for use with optinist demo on the cloud

Reply:
As noted above, it is better to prepare a similar script for building a local environment, but we assume it is OK here.